.github/workflows/e2e_openssl.yml

```diff
@@ -61,3 +63,7 @@ jobs:
       - name: E2E Test
         run: |
           nix shell .#contrast.e2e --command openssl.test -test.v workspace/just.containerlookup
+      - name: Cleanup
+        if: cancelled() && inputs.skip-undeploy != 'true'
```

Comment:
Can we please make `skip-undeploy` a boolean? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype